### PR TITLE
e2e DateTransform Plugin related testcases

### PR DESCRIPTION
**Pull request by Swati10-11**
Please review @itsankit-google @rmstar 

@sawantpritam

### CODE REVIEW

File: e2e-test/src/test/resources/pluginDataCyAttributes.properties [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+updateTableSchema=switch-allowSchemaRelaxation
+updateTableSchemaMacroInput=allowSchemaRelaxation
+outputSchemaMacroInput=Output Schema-macro-input
+path=path
+pathSuffix=suffix
+format=select-format
+formatMacroInput=format
+sampleSize=sampleSize
+delimiter=delimiter
+skipHeader=switch-skipHeader
```
Comment (rmstar) [inline e2e-test/src/test/resources/pluginDataCyAttributes.properties]: please remove configs that are not relevant to date transform plugin

File: e2e-test/src/test/resources/pluginParameters.properties [hunk 2]
```diff
@@ -0,0 +1,18 @@
+projectId=cdf-athena
+dataset=test_automation
+csvFormat=csv
```
Comment (rmstar) [inline e2e-test/src/test/resources/pluginParameters.properties]: please remove irrelevant configs

**Review by rmstar — COMMENTED** [review]
Date transform plugin needs to be downloaded from the hub. How are we handling that step when running using Github actions?

File: e2e-test/src/test/resources/pluginDataCyAttributes.properties (hunk 1 above)
Comment (Swati10-11) [reply e2e-test/src/test/resources/pluginDataCyAttributes.properties]: Removed

File: e2e-test/src/test/resources/pluginParameters.properties (hunk 2 above)
Comment (Swati10-11) [reply e2e-test/src/test/resources/pluginParameters.properties]: Removed

**Comment by itsankit-google** [comment]
> Date transform plugin needs to be downloaded from the hub. How are we handling that step when running using Github actions?

Using the `run_e2e_test` script, we deploy the latest build plugin, why will we need to download it from hub?
https://github.com/cdapio/cdap-e2e-tests/blob/develop/src/main/scripts/run_e2e_test.py#L70

**Review by itsankit-google — COMMENTED** [review]
These tests should be added in respective plugin modules instead of the parent? @ksahilCF

**Comment by Swati10-11** [comment]
New PR raised
https://github.com/data-integrations/date-transform/pull/15

**Pull request closed without merging**